### PR TITLE
Misc cleanups to fwd.go

### PR DESCRIPTION
**Pull request by DanG100**
* implement  gRPC server
* remove logging

### CODE REVIEW

File: dataplane/forwarding/fwd.go [hunk 1]
```diff
@@ -124,54 +192,111 @@ func (f *Engine) UpdatePacketSink(contextID *fwdpb.ContextId, packet fwdcontext.
...
+			if err = e.UpdatePacketSink(cid, ps, paddr); err != nil {
+				return nil, err
+			}
+
+		case fwdpb.ContextUpdateRequest_OPERATION_UPDATE_NOTIFICATION_ADDRESS:
+			if err = e.UpdateNotification(cid, ns, naddr); err != nil {
+				return nil, err
+			}
+		}
+	}
```
Comment (wenovus) [inline dataplane/forwarding/fwd.go:260]: Should the second for loop just be merged with the first? It seems to be calling update multiple times so it's confusing what the intention of the first for loop is.

File: dataplane/forwarding/fwd.go [hunk 2]
```diff
@@ -56,11 +60,18 @@ import (
 // forwarding contexts, each of which contain forwarding objects such as
 // tables, ports and actions.
 type Engine struct {
+	fwdpb.UnimplementedServiceServer
+	fwdpb.UnimplementedInfoServer
+	fwdpb.UnimplementedPacketSinkServer
+
```
Comment (wenovus) [inline dataplane/forwarding/fwd.go:66]: (no action required) it looks like you're now making this Engine implement the server where it did not before. I'm curious where this Engine was used prior to this refactoring?

File: dataplane/forwarding/fwd.go (hunk 1 above)
Comment (DanG100) [reply dataplane/forwarding/fwd.go]: good catch

File: dataplane/forwarding/fwd.go (hunk 2 above)
Comment (DanG100) [reply dataplane/forwarding/fwd.go:66]: there was a separate struct that implement in RPC interface and it called the Engine's  funcs, I just simplified it

Comment (wenovus) [reply dataplane/forwarding/fwd.go:66]: thanks for clarifying.

**Review by wenovus — APPROVED** [review]

**Pull request merged**